modules/signal-r-service/web-pub-sub/main.bicep

```diff
@@ -30,11 +30,8 @@ param capacity int = 1
 @description('Optional. Pricing tier of the resource.')
 param sku string = 'Standard_S1'
 
-@description('Optional. Enables system assigned managed identity on the resource.')
-param systemAssignedIdentity bool = false
-
-@description('Optional. The ID(s) to assign to the resource.')
-param userAssignedIdentities object = {}
+@description('Optional. The managed identity definition for this resource.')
```

Comment:
I wonder if we should mention that it can only be either or

Reply:
Added it to all modules like this (commit https://github.com/Azure/ResourceModules/pull/4124/commits/eb53c23b5848180c48399a826001d861f0588bfc). If you have suggestions regarding the wording, please let me know.